swift/example_code/ddb/README.md

```diff
@@ -0,0 +1,64 @@
+# DynamoDB code examples for the SDK for Swift
+## Overview
+This folder contains code examples demonstrating how to use the AWS SDK for
+Swift to use Amazon DynamoDB. This README discusses how to run these examples.
+
+Amazon DynamoDB is a fully managed NoSQL database service that provides fast
+and predictable performance with seamless scalability. DynamoDB lets you
+offload the administrative burdens of operating and scaling a distributed
+database so that you don't have to worry about hardware provisioning, setup
+and configuration, replication, software patching, or cluster scaling.
+
+## ⚠️ Important
+* Running this code might result in charges to your AWS account. 
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/DynamoDB/latest/UserGuide/best-practices.html#grant-least-privilege). 
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Code examples
+
+### Single actions
+Code excerpts that show you how to call individual service functions.
+* [TITLE](PATH TO SOURCE FILE WITH DDB CALLS)
```

Comment:
You need an entry here for each of the actions in the scenario, and then point to the appropriate *.swift file.